bots/images/scripts/fedora-28.bootstrap

```diff
@@ -18,4 +18,4 @@
 # 02110-1301 USA.
 
 BASE=$(dirname $0)
-$BASE/virt-install-fedora "$1" x86_64 "http://dl.fedoraproject.org/pub/fedora/linux/releases/28/Server/x86_64/os/"
+$BASE/virt-install-fedora "$1" x86_64 "http://dl.fedoraproject.org/pub/fedora/linux/releases/29/Server/x86_64/os/"
```

Comment:
Don't change the fedora-28.bootstrap script, just change the fedora-testing.bootstrap symlink to point to fedora-29.bootstrap. :-)

Reply:
ups, good catch.